server/package.json

```diff
@@ -1,21 +1,26 @@
 {
   "name": "ajo-backend",
   "version": "0.0.1",
-  "main": "index.js",
+  "main": "app.ts",
   "license": "MIT",
   "scripts": {
     "prepare": "cd .. && husky install server/.husky",
     "lint": "echo $PWD",
-    "dev": "nodemon ./dist/app.js",
+    "dev": "cross-env NODE_ENV=development nodemon ./dist/app.js",
+    "prod": "cross-env NODE_ENV=production nodemon ./dist/app.js",
```

Comment:
the db is setup in such a way that we can easily and dynamically switch between **development mode** in which we using  our local db on our machine and **production mode** in which we are using a cloud db(elephant sql).
the cross-env is used for setting the node environment variable(NODE_ENV) in the package.jason.  Of course this can be achieved without installing cross-env, but the command is different for various OS like windows and mac, hence the need for cross-env.
cross-env works across all platforms.

Reply:
I see your idea. We can also achieve this with the right .env file

.env
.env.development
.env.production

Reply:
it'll look something like this

<img width="300" alt="image" src="https://github.com/Ajo-Savings-Project/ajo-savings/assets/48376769/7ef640de-9ad4-494b-aab6-153fea4d7913">